webpush.go

```diff
@@ -40,6 +40,8 @@ type Options struct {
 	HTTPClient      HTTPClient // Will replace with *http.Client by default if not included
 	Subscriber      string     // Sub in VAPID JWT token
 	TTL             int        // Set the TTL on the endpoint POST request
+	Urgency         string     // Set the Urgency header to change a message priority (Optional)
```

Comment:
Since there are only a few options could you move this to a type? As well as define the options as constants?

Something similar to this would do:
```go
type Urgency string

const (
  UrgencyVeryLow Urgency = "very-low"
  ...
)
```